crates/steel-core/src/rvals.rs

```diff
@@ -836,6 +836,7 @@ pub enum SerializableSteelVal {
     MutFunc(MutFunctionSignature),
     HashMapV(Vec<(SerializableSteelVal, SerializableSteelVal)>),
     ListV(Vec<SerializableSteelVal>),
+    Pair(Box<(SerializableSteelVal, SerializableSteelVal)>),
```

Comment:
I had to add this new variant because of some [failing test](https://github.com/mattwparas/steel/actions/runs/9490147478/job/26152981083). I can't reproduce locally though :thinking:

Reply:
Interesting that you weren't able to reproduce locally, but I think this change is correct. When creating a new thread we have to serialize the value since each thread manages its own memory since nothing is really thread safe at the moment.

Reply:
I just tried to reproduce in my fork using the same GH actions and nothing... :thinking: